### PR TITLE
add develop mode that file change take effect immediate

### PR DESCRIPTION
**Pull request by mqshen**
add develop mode for static file(like css, js) change take effect immediate.

### CODE REVIEW

File: src/main/scala/com/earldouglas/xwp/ContainerPlugin.scala [hunk 1]
```diff
@@ -21,6 +22,8 @@ object ContainerPlugin extends AutoPlugin {
     val containerShutdownOnExit = settingKey[Boolean]("shutdown container on sbt exit")
 
     val containerLaunchCmd      = taskKey[Seq[String]]("command to launch container")
+
+    val runtimeClassPathCmd     = (fullClasspath in Runtime) map { cp => cp.map(_.data) }
```
Comment (Milyardo) [inline src/main/scala/com/earldouglas/xwp/ContainerPlugin.scala]: use the cp.files as shorthand for cp.map(_.data)

File: src/main/scala/com/earldouglas/xwp/ContainerPlugin.scala [hunk 2]
```diff
@@ -46,6 +49,7 @@ object ContainerPlugin extends AutoPlugin {
       Seq(libraryDependencies ++= (containerLibs in conf).value.map(_ % conf)) ++
       inConfig(conf)(Seq(
         start              := (startTask dependsOn webappPrepare).value
+      , develop            := (developTask dependsOn webappPrepare).value
```
Comment (Milyardo) [inline src/main/scala/com/earldouglas/xwp/ContainerPlugin.scala]: Calling `webappPrepare` is going to be confusing/a source of bugs by packaging dependencies into the webapp only for them to be in the system classpath instead. Infact depending on the webapp classloader delegation model this won't work at all.

A new webappPrepare task that doesn't package dependencies into to `WEB-INF/lib`/`WEB-INF/classes` is necessary as well.

Comment (mqshen) [inline src/main/scala/com/earldouglas/xwp/ContainerPlugin.scala]: I add runtime classPath manual when I start container.

Comment (Milyardo) [inline src/main/scala/com/earldouglas/xwp/ContainerPlugin.scala]: My point was that some application containers(tomcat in particular) will load resources from the webapp first, system classpath second. 

The develop task will be effectively broken in tomcat's default setting, unless you either change the setting for webapp classloader delegation, or create a webappPrepare task that doesn't package the dependencies into the webapp at all.

The later is option is more efficient, and portable.

**Comment by nafg** [comment]
Is this going to go anywhere?

**Comment by earldouglas** [comment]
@nafg I think this is a great idea in theory, but I'm not sure this particular implementation will work out.

**Comment by earldouglas** [comment]
Closing in favor of #319.

**Pull request closed without merging**